src/sentry/auth/access.py

```diff
@@ -271,6 +274,17 @@ class NoAccess(BaseAccess):
 
 
 def from_request(request, organization=None, scopes=None):
+    """
+    Returns the an Access object corresponding to the permissions associated with this request
+    and a particular organization. The user of the request could be an actual user
+    or it could be a sentry app
+    """
+    # TODO: validate scopes argument is not used and then remove
+    if scopes is not None:
+        import traceback
+
+        logger.info("from_request.non_none_scopes: %s" % traceback.format_exc())
```

Comment:
Might be safer to start with a metric?

Reply:
@EvanPurkhiser because of the log volume?

Reply:
just to get a quick judgment, and then you can add the logs after you verify it's not being used a lot.

Though if you're already sure logging is fine.

Reply:
@EvanPurkhiser I'm 90% sure we won't ever see the log messages at all.